content/ko/docs/concepts/overview/components.md

```diff
@@ -13,7 +13,7 @@ card:
 ---
 
 <!-- overview -->
-쿠버네티스를 배포하면 클러스터를 얻는다.
+쿠버네티스를 배포하면 클러스터가 생성된다.
```

Comment:
Why do you think that the revision is better? "Try not to use awkward passive voice when translating." is generally recommended practical suggestion for Korean translation.